### ISSUE TITLE
Add an option to download releases.

### ISSUE THREAD

**Issue by glennpratt** [github/hub#1102]
[feature request]
I see I can list releases now, but I cannot download them from private repositories with just the URLs.

```
~/bin/hub release show 0.0.0-gpratt4 --show-downloads


## Downloads

https://github.com/topsecretorg/foobar/releases/download/0.0.0-gpratt4/foobar-linux-3.19.0-30-generic-x86_64-0.0.0-gpratt4.tar.gz
https://api.github.com/repos/topsecretorg/foobar/zipball/0.0.0-gpratt4
https://api.github.com/repos/topsecretorg/foobar/tarball/0.0.0-gpratt4
```

**Comment by glennpratt** [comment]
See #1103

**Comment by glennpratt** [comment]
Merged in e046b8d

**Comment by mislav** [comment]
@glennpratt Thanks for following up by closing this. In the future, you can use the phrase `Fixes/closes #123` in your pull request body. That will automatically close the referenced issue when the PR is merged! :wink:
